components/breadcrumb/stories/breadcrumb.test.js

```diff
@@ -3,36 +3,153 @@ import { Template } from "./template.js";
 
 export const BreadcrumbGroup = Variants({
 	Template,
-	testData: [{
-		testHeading: "Default"
-	}, {
-		testHeading: "Compact",
-		variant: "compact",
-	},
-	{
-		testHeading: "Multiline",
-		variant: "multiline",
-		items: [
-			{
-				label: "Nav root",
-			},
-			{
-				iconName: "FolderOpen",
-				isDisabled: true,
-			},
-			{
-				label: "Trend",
-				isDragged: true,
-			},
-			{
-				label: "January 2019 Assets",
-			},
-		],
-	}],
+	testData: [
+		{
+			testHeading: "Default",
+		},
+		{
+			testHeading: "Default (nested)",
+			withStates: false,
```

Comment:
Why no states with nesting? Wouldn't it give us a chance to see if active or focus state styles are overlapping content or misaligned?

Reply:
There weren't any active or focus styles in the testing grid previously, so I didn't have those in my head when I was writing these- excellent point, though. I had created a separate set of `items` for that dragged/disabled view, so everything would have been identical, which is why I wasn't including the states. But that doesn't make much sense (to have a separate dragged/disabled `items`, identical for each variant) saying it now, so regardless, I need to refactor.

This now include the dragged/disabled states for each story. I tried to do a little clean up, too. https://github.com/adobe/spectrum-css/pull/2793/commits/6661742634de5de7b783fd9109a81ee79ade80a3

As an additional question- want me to make a new card to audit and capture active/hover/focus/etc states in components that should have those states?